release-notes/v3.29.0-release-notes.md

```diff
@@ -1,94 +1,96 @@
 28 Oct 2024
 
-#### Headline feature 1
+#### AdminNetworkPolicy support
 
-#### Headline feature 2
+- Add support for AdminNetworkPolicy egress network CIDRs. [calico 9303](https://github.com/projectcalico/calico/pull/9303) (@mazdakn)
+- Allow users to specify the default action in a tier. [calico 9245](https://github.com/projectcalico/calico/pull/9245) (@mazdakn)
+- Add support for the core functionalities of the  new k8s AdminNetworkPolicy API. [calico 9206](https://github.com/projectcalico/calico/pull/9206) (@mazdakn)
+- Tiers: A tier's order is by default 1,000,000. A nil order similarly is treated as 1M. [calico 9189](https://github.com/projectcalico/calico/pull/9189) (@mazdakn)
```

Comment:
Not sure this one really need a release note? And if it does, I don't think this makes much sense to users reading this - they don't even know what tiers are yet!

Reply:
good point! thanks!